src/wms/src/main/java/org/geoserver/wms/WMS.java

```diff
@@ -1273,6 +1275,15 @@ public TreeSet<Date> getFeatureTypeTimes(FeatureTypeInfo typeInfo) throws IOExce
         FeatureCollection collection = getDimensionCollection(typeInfo, time);
 
         TreeSet<Date> result = new TreeSet<>();
+
+        String startValue = time.getStartValue();
+        String endValue = time.getEndValue();
+        if (!StringUtils.isEmpty(startValue) && !StringUtils.isEmpty(endValue)) {
```

Comment:
The configuration can be changed via UI (which I see is resetting the fields contents if list presentation is chosen), but can also be modified via REST, often without full understanding of how things are supposed to work. I'd recommend adding a ``&& time.getPresentation != DimensionPresentation.LIST``, or similar condition, to make sure the presentation and configured values are compatible.